main.go

```diff
@@ -4,6 +4,7 @@ import (
 	"log"
 
 	"github.com/netlify/gotrue/cmd"
+	_ "github.com/netlify/gotrue/docs"
```

Comment:
This is also concerning a bit... is this really necessary? Can't we put a `main` function inside `docs` that does nothing -- if that's important to the tool.